sdk/monitor/Azure.Monitor.Ingestion/tests/SessionRecords/MonitorIngestionLiveTest/ConcurrencyMultiThreadAsync.json

```diff
@@ -1,180 +1,180 @@
-        "x-ms-request-id": "0d67299b-5219-4c19-bb36-0af92e55ba40"
+        "x-ms-client-request-id": "abdc147d-ba7a-eba7-b74c-902c74a99d4d",
+        "x-ms-request-id": "31a3377d-b5e2-46f8-bbcc-5f623654673b"
       },
       "ResponseBody": null
     },
     {
-      "RequestUri": "https://98c96978-a857-58f1-a299-7d216d4b043a-p2j8.westus-1.ingest.monitor.azure.com/dataCollectionRules/dcr-37a02bb9a8ba4f97a80716652c78e678/streams/Custom-MyTableRawData?api-version=2021-11-01-preview",
+      "RequestUri": "https://86923540-75e9-5143-84b6-7535316a68dc-lipn.westus-1.ingest.monitor.azure.com/dataCollectionRules/dcr-ba536bf3cee94b1db8d04b30ee3f2e80/streams/Custom-MyTableRawData?api-version=2021-11-01-preview",
       "RequestMethod": "POST",
       "RequestHeaders": {
         "Accept": "application/json",
         "Authorization": "Sanitized",
         "Content-Encoding": "gzip",
-        "Content-Length": "85",
+        "Content-Length": "86",
         "Content-Type": "application/json",
-        "traceparent": "00-0feae899d9b5ac4f8bb76518014d8cfe-0b59ba6047d8e342-00",
-        "User-Agent": "azsdk-net-Monitor.Ingestion/1.0.0-alpha.20221012.1 (.NET Core 3.1.30; Microsoft Windows 10.0.22621)",
-        "x-ms-client-request-id": "c198e9bf851fb6c800167fd3db0e9149",
+        "traceparent": "00-84024e655bda424a8663ab11d00b1efd-8b499e975c06aa4c-00",
+        "User-Agent": "azsdk-net-Monitor.Ingestion/1.0.0-alpha.20221024.1 (.NET Core 3.1.30; Microsoft Windows 10.0.22621)",
+        "x-ms-client-request-id": "644bc58d4041e71d94379271b0646ad3",
         "x-ms-return-client-request-id": "true"
       },
-      "RequestBody": "[{\u0022Time\u0022:\u00222022-10-13T14:10:38.720123\u0022,\u0022Computer\u0022:\u0022Computer1\u0022,\u0022AdditionalContext\u0022:1}]]",
+      "RequestBody": "[{\u0022Time\u0022:\u00222022-10-24T17:04:30.6669136\u0022,\u0022Computer\u0022:\u0022Computer4\u0022,\u0022AdditionalContext\u0022:4}]]",
       "StatusCode": 204,
       "ResponseHeaders": {
-        "Date": "Thu, 13 Oct 2022 21:10:38 GMT",
+        "Date": "Tue, 25 Oct 2022 00:04:31 GMT",
         "Server": "Microsoft-HTTPAPI/2.0",
-        "x-ms-client-request-id": "c198e9bf-851f-b6c8-0016-7fd3db0e9149",
-        "x-ms-request-id": "cca18c51-0094-44f4-9d89-9d1eff035660"
+        "x-ms-client-request-id": "644bc58d-4041-e71d-9437-9271b0646ad3",
+        "x-ms-request-id": "17d2fd51-1590-4053-8485-ef92f4d59bc1"
       },
       "ResponseBody": null
     },
     {
-      "RequestUri": "https://98c96978-a857-58f1-a299-7d216d4b043a-p2j8.westus-1.ingest.monitor.azure.com/dataCollectionRules/dcr-37a02bb9a8ba4f97a80716652c78e678/streams/Custom-MyTableRawData?api-version=2021-11-01-preview",
+      "RequestUri": "https://86923540-75e9-5143-84b6-7535316a68dc-lipn.westus-1.ingest.monitor.azure.com/dataCollectionRules/dcr-ba536bf3cee94b1db8d04b30ee3f2e80/streams/Custom-MyTableRawData?api-version=2021-11-01-preview",
```

Comment:
In general, we shouldn't need to re-record tests whenever a change is made. Was there a mismatch in playback that made it necessary to re-record?